SingularityService/src/main/java/com/hubspot/singularity/config/UIConfiguration.java

```diff
@@ -59,6 +59,10 @@ public static RootUrlMode parse(String value) {
   @NotNull
   private List<ShellCommandDescriptor> shellCommands = Collections.emptyList();
 
+  @JsonProperty
+  @NotNull
+  private Boolean showTaskDiskSpace = false;
```

Comment:
also, `Boolean` is nullable while `boolean` is not (because it's a primitive type) -- better to not have to deal with `null` values when you don't need to